src/lib/SideBar/SideBarMenu.tsx

```diff
@@ -1,12 +1,18 @@
 import { useSideBarMenuContext } from "./SideBarMenuContext";
 import { SideBarMenuItem } from "./SideBarMenuItem";
 
-const SideBarMenu = () => {
+interface SideBarMenuProps {
+  theme: "dark" | "light";
+}
+
+const SideBarMenu = ({ theme }: SideBarMenuProps) => {
   const { items } = useSideBarMenuContext();
 
+  const sidenavClass = theme == "dark" ? "sb-sidenav-dark" : "";
```

Comment:
I think this would be slightly better:

```suggestion
  const sidenavClass = `sb-sidenav-${theme}`;
```

Just in case a consumer of the package wants to add some css styles to `sb-sidenav-light`. 

What's you opinion? @fwermelinger

Reply:
I had this first, but then read that the original documentation says: if you want ligh theme, use just `sb-sidenav`. If you want dark theme add `sb-sidenav-dark` so that's what I tried to follow. But I also think it's better to have the class.